lib/ansible/module_utils/gcp.py

```diff
@@ -350,6 +360,17 @@ def get_google_cloud_credentials(module, scopes=None):
         return (None, None)
 
 
+def gcp_storage_connect(module, credentials, conn_params, scopes=None):
+    if not HAS_GOOGLE_STORAGE_LIB:
+        module.fail_json(msg="Please install google-cloud-storage to run this module")
+    if not scopes:
+        scopes = GOOGLE_STORAGE_DEFAULT_SCOPES
```

Comment:
Can you set GOOGLE_STORAGE_DEFAULT_SCOPES as the default parameter in line 363?

Reply:
It seems this cannot be done since mutable objects like builtin list or dictionaries are considered to be 'dangerous default values' for keyward args. This, in turn, results into a pylint failure.
Hence, would like to stay back on this change.
Please let me know if you would like to have any further modifications to this.